### PR TITLE
doc/1212_add_path_creations_and_permissions

### PR DESCRIPTION
**Pull request by pcoello25**
* Fix issue  #1212 
* No tests are required
* Assignee: @frbattid

### CODE REVIEW

File: doc/cygnus-common/installation_and_administration_guide/install_from_sources.md [hunk 1]
```diff
@@ -49,6 +50,16 @@ Once created, change to this new fresh user in order to proceed with the rest of
 
 [Top](#top)
 
+###<a name="section2.1"></a>`log4j` path
+Once the user is created is necessary to create the path `/var/log/cygnus` due to `log4j` purposes. Starting by creating the path and then give permissions for `cygnus` user:
```
Comment (frbattid) [inline doc/cygnus-common/installation_and_administration_guide/install_from_sources.md]: for `log4j` purposes
Start by creating

File: doc/cygnus-common/installation_and_administration_guide/install_from_sources.md [hunk 2]
```diff
@@ -49,6 +50,16 @@ Once created, change to this new fresh user in order to proceed with the rest of
...
+
+    $ mkdir -p /var/log/cygnus
+    $ chown -R cygnus:cygnus /var/log/cygnus
+
+This step is important because if you don't have the log path created Cygnus is going to shutdown when running first time.
```
Comment (frbattid) [inline doc/cygnus-common/installation_and_administration_guide/install_from_sources.md]: Cygnus will shutdown when running for the first time

Comment (pcoello25) [inline doc/cygnus-common/installation_and_administration_guide/install_from_sources.md]: Fixed in 5fcab1c

File: doc/cygnus-common/installation_and_administration_guide/install_from_sources.md (hunk 1 above)
Comment (pcoello25) [inline doc/cygnus-common/installation_and_administration_guide/install_from_sources.md]: Fixed in 5fcab1c

**Pull request merged**